src/Middlewares/Validation/index.js

```diff
@@ -0,0 +1,111 @@
+import Joi from "joi";
+
+export default class Validation {
+  static async blogFormValidation(req, res, next) {
+    const SCHEMA = Joi.object({
+      BlogTitle: Joi.string().min(3).required(),
+      BlogContent: Joi.string().min(120).required(),
+      BlogPic: Joi.string().uri(),
+    });
+    try {
+      const { error } = SCHEMA.validate(req.body);
+      if (error)
+        return res.status(400).json({
+          Error: error.details[0].message,
+        });
+      else next();
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  static async blogUpdateFormValidation(req, res, next) {
```

Comment:
Please put the `user validation functions` in a separate file, and `blog validation functions` in another separate file.